guides/ansible_engine/README.md

```diff
@@ -0,0 +1,33 @@
+# Ansible Lightbulb - Ansible Engine
+
+This guide is part of lightbulb, a multi-purpose toolkit for effectively demonstrating Ansible's capabilities or providing informal workshop training in various forms -- instructor-led, hands-on or self-paced.
+
+## Presentation
+The presentation deck accompanying this course can be found at:
+[Ansible Essentials Lightbulb Deck](../../decks/ansible-essentials.html)
+
+## Ansible Engine Exercises
+
+This guide consits of multiple exercises, it is strongly recommended to run them one after the other. They are self-explaining and can be run at self pace.
+
+ - [Exercise 1 - Running Ad-hoc commands](1-adhoc)
+ - [Exercise 2 - Writing Your First playbook](2-playbook)
+ - [Exercise 3 - Using Variables, Loops, and Handlers](3-variables)
+ - [Exercise 4 - Running the apache-basic-playbook](4-runplaybook )
+ - [Exercise 5 - Roles: Making your playbooks reusable](5-roles)
+
+## Survey
+We are always keen for feedbacl. If you attend an in-person workshop, please fill out the survey!
+-> [Click for Survey](http://bit.ly/net-lightbulb-survey) <-
```

Comment:
Hum. What's this? Seems network automation and Red Hat specific. Will need to run this by marketing.

Reply:
I didn't knew, I thought it is lightbulb specific, not networking specific. I will remove that for now.